zulipterminal/cli/run.py

```diff
@@ -92,7 +92,7 @@ def parse_args(argv: List[str]) -> argparse.Namespace:
                         action="store_true",
                         help='list all the color themes.')
     parser.add_argument('--color-depth',
-                        choices=['1', '16', '256'],
+                        choices=['1', '16', '256', '2**24'],
```

Comment:
Note that the new `color-depth` option needs to be wrapped in quotes when passing to the CLI parameter. This is not the case with other options, which makes the new one inconsistent and a bit long compared to other options.

Alternatively, we could use something like `2^24` which is shorter and don't need to be wrapped in quotes?

Reply:
Resolved as `24bit`